src/main/java/org/junit/Assume.java

```diff
@@ -34,6 +34,15 @@
  * @since 4.4
  */
 public class Assume {
+
+    /**
+     * Do not instantiate.
+     * @deprecated will be private soon.
```

Comment:
"soon" seems too vague to be useful.  I think I'd prefer "deprecated since 4.13"; if people want to know what that means, they can look up the deprecation policy.